scripts/dev-restart.sh

```
@@ -0,0 +1,172 @@
+        fi
+    else
+        echo "✅ Port $port is already free"
+    fi
+}
+
+# Function untuk load environment variables
+load_env() {
+    local env_file=$1
+    if [ -f "$env_file" ]; then
+        echo "📄 Loading environment from $env_file"
+        export $(grep -v '^#' "$env_file" | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)